llvm/test/Transforms/SLPVectorizer/RISCV/ctpop.ll

```diff
@@ -0,0 +1,115 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv32 -mattr=+m,+v | FileCheck %s --check-prefixes=CHECK,RV32
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+m,+v | FileCheck %s --check-prefixes=CHECK,RV64
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv32 -mattr=+v,+experimental-zvbb | FileCheck %s --check-prefixes=CHECK,ZVBB-RV32
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+v,+experimental-zvbb | FileCheck %s --check-prefixes=CHECK,ZVBB-RV64
```

Comment:
Drop unused check labels and remove extra comments at the end of the file, generated by the script